nucypher/blockchain/eth/networks.py

```diff
@@ -30,18 +30,22 @@ class NetworksInventory:  # TODO: See #1564
     UNKNOWN = 'unknown'  # TODO: Is there a better way to signal an unknown network?
     DEFAULT = MAINNET
 
-    __to_ethereum_chain_id = {
+    __to_ethereum_chain_id_eth = {
```

Comment:
I don't think there are any inter-chain ID collisions, why use two dictionaries?

Reply:
The eventual split tuple of `ETH_NETWORKS` and `POLY_NETWORKS` allows for easier validation, and selection listing in the CLI - https://github.com/theref/nucypher/blob/ursula-init/nucypher/cli/actions/select.py#L139.

Did you have an alternative thought here?

Reply:
Well in any case `__to_ethereum_chain_id_eth` is a bit redundant 🤔